#constant

uint8 FPV_CAMERA  = 7
uint8 MAIN_CAMERA = 0
uint8 VICE_CAMERA = 1
uint8 TOP_CAMERA  = 2

#request

bool is_h264
bool is_open
uint8 request_view
---
#reponse

uint8[] raw_data
uint32  raw_data_len
uint16  height
uint16  width
bool result
